hoomd/hpmc/pytest/test_external.py

```diff
@@ -0,0 +1,188 @@
+# Copyright (c) 2009-2021 The Regents of the University of Michigan
+# This file is part of the HOOMD-blue project, released under the BSD 3-Clause
+# License.
+
+"""Test hoomd.hpmc.external.field."""
+
+import hoomd
+import pytest
+import numpy as np
+
+valid_constructor_args = [
+    dict(reference_positions=[[0, 0, 0]],
+         reference_orientations=[[1, 0, 0, 0]],
+         k_translational=1.0,
+         k_rotational=1.0,
+         symmetries=[[1, 0, 0, 0]]),
+]
```

Comment:
If we are only going to test one combination of parameters no need to make the test parametrized. If we wanted to test multiple combinations then that would make sense.

Reply:
Added more sets of parameters to test.